kaprekar.java

```diff
@@ -0,0 +1,54 @@
+// Java program to check if a number is  
+// Kaprekar number or not 
+  
+class GFG  
```

Comment:
The filename and the class name should have to be same otherwise the code won't run. Please update the class name.

And, make sure to use Came casing while renaming the file as well as the class name.

Reply:
thanks  for letting me know.I have done the required adjstment